styles/mixins/triangle.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function( config ) {
```

Comment:
We could still probably use CSS transforms to make this possible with the current mixin if need be.